packages/inter-protocol/src/psm/psm.js

```diff
@@ -1,9 +1,6 @@
 // @jessie-check
 /// <reference types="@agoric/governance/exported" />
-
-// XXX ambient types runtime imports until https://github.com/Agoric/agoric-sdk/issues/6512
-import '@agoric/zoe/exported.js';
-import '@agoric/zoe/src/contracts/exported.js';
+/// <reference types="@agoric/zoe/exported" />
```

Comment:
:clap: 

I'm curious about the impact on bundle sizes for these contracts.

Reply:
**before**
```
    22135 bundle-econCommitteeCharter-js-meta.json
  1166313 bundle-econCommitteeCharter.js
    28423 bundle-fluxAggregatorKit-js-meta.json
  1534301 bundle-fluxAggregatorKit.js
    28121 bundle-psm-js-meta.json
  1496517 bundle-psm.js
```

**after**
```
    20614 bundle-econCommitteeCharter-js-meta.json
  1105853 bundle-econCommitteeCharter.js
    26868 bundle-fluxAggregatorKit-js-meta.json
  1474173 bundle-fluxAggregatorKit.js
    26566 bundle-psm-js-meta.json
  1435797 bundle-psm.js
```

| bundle | savings | 
| -- | -- | 
| psm | 60,720 |
| fluxAggregatorKit | 98,504 | 
| econCommitteeCharter | 60,460 |

Reply:
at [2844e7c](https://github.com/Agoric/agoric-sdk/pull/9373/commits/2844e7c950f6486b0ed753a7daba9dcba6291fbc)

```
  1104273 bundle-econCommitteeCharter.js
  1472593 bundle-fluxAggregatorKit.js
  1434213 bundle-psm.js
```

| bundle | abs drop |  rel drop |
| -- | -- | --|
| econCommitteeCharter | 62,040 | 5.3%
| fluxAggregatorKit | 61,708 | 4.0%
| psm | 60,720 |  4.0%